lists/in.csv

```diff
@@ -427,7 +426,7 @@ https://www.facebook.com/groups/346504835433106/?ref=ts,REL,Religion,2014-04-15,
 https://www.facebook.com/groups/349745378434446/,REL,Religion,2014-04-15,citizenlab,
 http://www.facebook.com/groups/410690962321650,REL,Religion,2014-04-15,citizenlab,
 http://www.facebook.com/groups/421881071186596/?ref=ts,REL,Religion,2014-04-15,citizenlab,
-http://www.facebook.com/groups/43700222634264\ad\ad3/?ref=ts,REL,Religion,2014-04-15,citizenlab,
+http://www.facebook.com/groups/43700222634264,REL,Religion,2014-04-15,citizenlab,
```

Comment:
That _probably_ should be https://www.facebook.com/groups/437002226342643
At least that's alive link. I'm not sure what `\ad` is.